### ISSUE TITLE
Can't download model

### ISSUE THREAD

**Issue by Kevoting** [TheLastBen/fast-stable-diffusion#2430]
Can't download custom model via MODEL_LINK
i get:
FileNotFoundError                         Traceback (most recent call last)
[<ipython-input-4-06e2bcf5d16c>](https://localhost:8080/#) in <cell line: 132>()
    174             model=f'/content/gdrive/{mainpth}/sd/'+nosd_name1+'-{blasphemy}/models/'+nosd_name2+'/{modelname}'
    175          if not os.path.exists(model):
--> 176             gdown.download(url=MODEL_LINK, output=model, quiet=False, fuzzy=True)
    177             clear_output()
    178          else:

[/usr/local/lib/python3.10/dist-packages/gdown/download.py](https://localhost:8080/#) in download(url, output, quiet, proxy, speed, use_cookies, verify, id, fuzzy, resume)
    217                     "Please remove them except one to resume downloading.",
    218                     file=sys.stderr,
--> 219                 )
    220                 return
    221             tmp_file = existing_tmp_files[0]

FileNotFoundError: [Errno 2] No such file or directory: '/content/gdrive/MyDrive/sd/stable-diffusion-{blasphemy}/models/Stable-diffusion'

**Comment by TheLastBen** [comment]
did you run the cells before ? do a screenshot of the model cell

**Comment by Kevoting** [comment]
Yes, it was working initially, but after some update it was broken
I think it's the {blasphemy} variable
![Screenshot_3](https://github.com/TheLastBen/fast-stable-diffusion/assets/32854411/74a41e9e-fe0a-4446-a811-646550aefbac)

**Comment by TheLastBen** [comment]
I'll check it out

**Comment by Kevoting** [comment]
It's the default

**Comment by TheLastBen** [comment]
`model=f'/content/gdrive/{mainpth}/sd/'+nosd_name1+'-{blasphemy}/models/'+nosd_name2+'/{modelname}'`

this line was modified, you need to change the lora download cell accordingly
